apps/ejabberd/src/mod_disco.erl

```diff
@@ -464,7 +464,7 @@ make_node_attr(Node) -> [{<<"node">>, Node}].
 
 -spec get_info(A :: [jlib:xmlel()], ejabberd:server(), module(), Node :: binary(),
         Lang :: ejabberd:lang()) -> [jlib:xmlel()].
-get_info(_A, Host, Mod, Node, _Lang) when Node == [] ->
+get_info(A, Host, Mod, Node, _Lang) when Node == <<>> ->
```

Comment:
A is two short.
Just by reading the second function clause I can understand that it's Acc.